### PR TITLE
book.html changes in the main

### PR DESCRIPTION
**Pull request by Prince-rj**
pull request changes in book.html

### CODE REVIEW

**Review by Prince-rj — COMMENTED** [review]
problems

File: css/books.css [hunk 1]
```diff
@@ -32,15 +36,128 @@ body{
...
+    width: 400px;
+    /* flex-wrap:wrap; */
+    justify-content: center;
+    align-items: center;
+    background-color: white;
 }
 .con1, .con1prim{
     display:flex;
-    margin-top: 23px;
+    /* margin-top: 23px; */
```
Comment (zekst) [inline css/books.css:160]: Can please remove unnecessary comments. Check for all the occurrences

**Review by zekst — COMMENTED** [review]
Hi @Prince-rj , Can you please add demo of after and before. It will help us to move forward with changes. Also remove the redundant comment and make sure you are not repeating your self.

**Review by Prince-rj — COMMENTED** [review]
changes required

**Pull request merged**